src/test/java/com/microsoft/sqlserver/jdbc/preparedStatement/BatchExecutionWithNullTest.java

```diff
@@ -110,6 +110,68 @@ public void testAddbatch2AEOnConnection() throws SQLException {
     public void testAddbatch2() throws SQLException {
         testAddBatch2(getConnection());
     }
+    
+    /**
+     * TestClearBatch with AE enabled on the connection
+     * 
+     * @throws SQLException
+     */
+    @Test
+    @Tag(Constants.xSQLv12)
+    public void testClearBatchAEOnConnection() throws SQLException {
+        try (Connection connection = PrepUtil.getConnection(connectionString + ";columnEncryptionSetting=Enabled;")) {
+            testClearBatch(connection);
+        }
+    }
+    
+    /**
+     * Test the same as testClearBatchAEOnConnection, with AE disabled
+     * 
+     * @throws SQLException
+     */
+    @Test
+    public void testClearBatch() throws SQLException {
+        testClearBatch(getConnection());
+    }
+    
+    public void testClearBatch(Connection conn) throws SQLException {
+        String batchTable = TestUtils
+                .escapeSingleQuotes(AbstractSQLGenerator.escapeIdentifier(RandomUtil.getIdentifier("batchTable")));
```

Comment:
all the tests in this class use  `tableName` it creates that for every test in `@BeforeEach` so this gets created here too. Can this use the same table? Otherwise we need to update `testSetup` and `terminateVariation`. Also, usually we do a `dropTableIfExists` before table creation in case there's junk left from previous runs.

Reply:
The table columns in the `testClearBatch` cases are different from the ones we use in other tests. It currently creates a new table when `testClearBatch` with and without AE run, and drops the table in the `finally` block. If we are going to use the same table name for each test, we need to move the table creation out of `@BeforeEach` and create each table in the corresponding test. What do you think?

Reply:
Added comment based on conversation.